### PR TITLE
Enable daily forecast

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -29,19 +29,19 @@ def generate_forecast():
...
     frequency = 'currently'  # default forecast set to current
-    valid_frequency = ['currently', 'hourly']
+    valid_frequency = ['currently', 'hourly', 'daily']
     last_query = place.split(' ')[-1].lower()
     if last_query in valid_frequency:
         place = place.replace(last_query, '')  # replace time method(if present) with empty string so that only the place remains. Example query: cebu hourly
         frequency = last_query
 
+    geocode = Geocode(address=place)
+    is_place_valid = geocode.is_place_query_valid()
```
Comment (mknecht) [inline app/views.py:40]: what does this have to do with daily forecast?

File: app/views.py [hunk 2]
```diff
@@ -71,13 +71,17 @@ def convert_to_readable_time(time):
...
     if frequency == 'currently':
-        return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.\
+        return '{}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.\
             format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
 
     info = []
     for count, item in enumerate(forecast.data):
-        info.append('time: {}'.format(convert_to_readable_time(item.time)))
-        info.append('temperature: {}'.format(item.temperature))
+        info.append('{}'.format(convert_to_readable_time(item.time)))
```
Comment (mknecht) [inline app/views.py]: @admiral96 why the '{}'format thing? it's pointless if there's nothing else in the format string. unless you want to just convert the value to string. then you should just use `str()`

File: test_views.py [hunk 3]
```diff
@@ -150,3 +150,52 @@ def test_valid_weather_url_valid_address_hourly(self, is_place_query_valid, get_
...
+                    "ozone": 241.73
+                }
+            ]
+        }
+        generate_forecast.return_value = forecastio.models.ForecastioDataBlock(d=mock_daily)
+        result = self.client.get('/weather_forecast?address=Lahug, Cebu City daily')
+        self.assertTrue(is_place_query_valid.called)
+        self.assertTrue(generate_forecast.called)
+        self.assertTrue(get_place_info.called)
+        self.assertEqual(result.status_code, 200)
```
Comment (mknecht) [inline test_views.py:213]: uh, can you please assert on the _forecast_? it's not enough that the server returns a 200. at least the temperature needs to be in there.

File: app/views.py (hunk 1 above)
Comment (admiral96) [inline app/views.py:40]: I only switched their position because I realized that it determines if the place is valid even without slicing out the frequency. What I did here is just to make sure that is slices first the frequency before validating the address.

Ex: Cebu, City daily
----------------------^
                         needs to extract this first then validate the address

Comment (admiral96) [inline app/views.py]: the 'daily' ones

Comment (mknecht) [inline app/views.py:40]: ah, sorry

**Comment by mknecht** [comment]
@admiral96 ok to merge
